logger/logger.go

```diff
@@ -0,0 +1,42 @@
+package logger
+
+import "fmt"
+
+var log *Logger
+
+// Logger provided simple logger.
+type Logger struct {
+	verbose bool
+}
+
+// Setup configures logger
+func Setup(verbose bool) *Logger {
```

Comment:
I don't think it's a good idea for this function to return the global instance of the logger as users might change the configuration of the instance they've received, which would, as a side-effect change the behavior of the global logger.

Instead, I suggest making this function void - ie, not returning any instance. And if, in the future, the need arises, we can introduce a `NewLogger` function which explicitly creates an instance of a logger, which can be configured as needed.